src/jsMain/kotlin/team/karakum/showcase/material/inputs/TextField.kt

```diff
@@ -17,13 +17,15 @@ import team.karakum.autoComplete
 import team.karakum.noValidate
 import web.cssom.Display
 import web.cssom.JustifyContent
+import web.cssom.em
 import web.html.InputType
 
 val TextFieldShowcase = FC<Props> {
     Box {
         sx {
             display = Display.grid
             justifyContent = JustifyContent.left
+            gap = 1.em
```

Comment:
Why here you don't use your common component?

Reply:
nailed it